src/dioptra/restapi/v1/models/service.py

```diff
@@ -65,6 +68,7 @@ def __init__(
         self,
         model_name_service: ModelNameService,
         group_id_service: GroupIdService,
+        mlflow_client: MlflowClient,
```

Comment:
As a first pass, I'm okay with injecting the third party `MlflowClient` directly, but as a technical debt follow-up issue I'd like to see its behavior wrapped similar to what we did with the `rq` library, where we make a dedicated class in the `shared` folder to create a standardized interface for the behaviors we want: https://github.com/usnistgov/dioptra/blob/71a646084266e379815d032fc48fea680e97d639/src/dioptra/restapi/v1/shared/rq_service.py

That also allows us to encapsulate logging and try/except handling in a single place that's outside of the services used directly in the endpoints.

So yeah, only action for this comment is to create an issue so we don't lose track of this.